.github/workflows/ipfs-deploy.yml

```diff
@@ -65,3 +65,71 @@ jobs:
       - name: Publish to nft.storage
         run: |
           docker run -e NFTSTORAGE_API_KEY=${{ secrets.NFTSTORAGE_API_KEY }} $IMAGE_RELEASE_ID nft.storage
+
+      - name: Create a release
+        if: startsWith(github.ref, 'refs/tags/')
+        env:
+          REF_NAME: ${{ github.ref_name }}
+          GITHUB_REPOSITORY: ${{ github.repository }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          # Markdown template for the release notes
+          RELEASE_NOTE_TEMPLATE=$(cat << EOF
+          #### IPFS Hash
+
+          \`\`\`
+          $IPFS_HASH
+          \`\`\`
+
```

Comment:
No need to put it on a line by itself, all of the links have it.
```suggestion
```

Reply:
I like that its clearly somewhere and you don't need to dig it on rest of the others

Reply:
Hmm, I guess having it be copyable (as you mentioned below) does give an advantage.  Will retract this suggestion.